TODO.md

```diff
@@ -7,6 +7,18 @@ Scratchpad for random ideas of things that need done.
 * Verify RDP enable
 * Linux repos
 * Whole buncha stuff for profile::cfcc::camper
+Charlie's $0.02
+
+* jdk8 > jdk10
+* remove eclipse
+* add a good text editor (I use notepad2-mod; I think lots of the others are overkill and overwhelming, nbd though)
+* Add registry entry to allow edit of any file from context menu in classesroot/*/editinnotepad[Edit in Notepad]/command/notepad.exe %1
+* add python 2.7? I recall we use odd python stuff that has compatibility issues and thought that might be one; I'm okay leaving it out if it's not an issue
```

Comment:
So..... the ProgramArcadeGames tutorial uses Python3. However they provide 32-bit runtime and Pygame packages on the site. This has caused a ton of confusion since Pycharm defaults to 64-bit. What makes this more complicated is that PyCharm 2018.1.3 defaults to creating a venv for every project and NOT including site-packages. We need to make sure counselors are aware of this so they can check the right box when creating projects (there's a quick "include" button in project setup)

Reply:
Yeah, I saw that button when I set it up earlier today.